vignettes/ConvertingPharmacoSetToGDR.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: "Converting PharmacoSet Drug Response Data into gDR object" 
+author:
+- name: Jermiah Joseph
+  email: jermiah.joseph@uhn.ca
+- name: Bartosz Czech
+  email: bartosz.w.czech@gmail.com
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ConvertingPharmacoSetDrugResponseDataintogDRobject}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(PharmacoGx)
+library(gDR)
+library(gDRcore)
+library(gDRimport)
+```
+
+
+# Overview
+The `gDRimport` package is a part of the gDR suite. It helps to prepare raw drug response data for downstream processing. 
+It mainly contains helper functions for importing/loading/validating dose response data provided in different file formats. 
```

Comment:
Maybe instead of `in different file formats`, we can specify `from different scanner sources`